.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -38,48 +33,56 @@ variables:
   # benchmarks get changed to run on every PR)
   allow_failure: true
 
-trace-agent-v04-4cpus-normal_load-fixed_sps-macrobenchmarks:
+trace-agent-v04-2cpus-normal_load-fixed_sps:
   extends: .trace_agent_benchmarks
   variables:
-    TRACE_AGENT_VERSION: main
     TRACE_AGENT_ENDPOINT: v04
     TRACE_AGENT_CPUS: 2
```

Comment:
Let's discuss this in PR in benchmarking-platform repo, mb it is better to use `TRACE_AGENT_CPUS: 40-41` to simplify scripts.